site/docs/components/stepper-input/examples.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="stepper-input" exampleName="Default" >
+
+## Default
+
+Stepper input has a default value of 0, an increment value of 1 and a block value of 10, and you can only use it to accept numeric characters. A single click increases or decreases the value in the input by 1.
+
+</LivePreview>
+<LivePreview componentName="stepper-input" exampleName="Variants" >
+
+## Variants
+
+The component comes in two variants: primary and secondary. By default, the primary variant is selected. To switch to the secondary variant, use `variant="secondary"`. When choosing between the primary and secondary variants, consider the background color on which the component will be placed.
```

Comment:
Not sure about the terminology "the primary variant is selected"

Reply:
How about "By default, the primary variant is displayed."